extensions/wikia/VideoEmbedTool/js/VET.js

```diff
@@ -702,7 +702,17 @@ define('wikia.vet', [
 			// attach handlers - search
 			this.cachedSelectors.searchForm.submit(function (event) {
 				event.preventDefault();
-				var keywords = $(this).find('#VET-search-field').val();
+				var esc_map = {
+					'&': '&amp;',
+					'<': '&lt;',
+					'>': '&gt;',
+					'"': '&quot;',
+					"'": '&#39;'
```

Comment:
Isn't [`mw.html.escape`](https://doc.wikimedia.org/mediawiki-core/master/js/#!/api/mw.html-method-escape) available for use in here?

Reply:
good catch, I forgot about existence of that one. Thanks!